pkgs/unified_analytics/lib/src/event.dart

```diff
@@ -19,6 +19,28 @@ final class Event {
       : eventName = DashEvent.analyticsCollectionEnabled,
         eventData = {'status': status};
 
+  /// Event that is emitted when an error occurs within
```

Comment:
start dartdocs with a single sentence summary separated by a line from the rest: https://dart.dev/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary